source/guide_adminer.rst

```diff
@@ -0,0 +1,140 @@
+.. highlight:: console
+
+.. author:: Nikolas <https://nikolasdas.de>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: database
+
+.. sidebar:: About
+
+  .. image:: _static/images/adminer.svg
+      :align: center
+
+#######
+Adminer
+#######
+
+.. tag_list::
+
+Adminer_ (formerly phpMinAdmin) is a full-featured database management tool written in PHP. Conversely to phpMyAdmin, it consist of a single file ready to deploy to the target server. Adminer is available for :manual:`MySQL <database-mysql>`, MariaDB, :lab:`PostgreSQL <guide_postgresql>`, SQLite, MS SQL, Oracle, Firebird, SimpleDB, Elasticsearch and :lab:`MongoDB <guide_mongodb>`.
+
+----
+
+.. note::
+
+  Uberspace provides instances of Adminer_ and phpMyAdmin_ for everbody to use. Therefore this guide is mainly for people who would like to customize Adminer to there needs.
+
+  * Adminer: https://mysql.uberspace.de/adminer/
+  * phpMyAdmin: https://mysql.uberspace.de/phpmyadmin/
+
+
+License
+=======
+
+Adminer_ is released under `Apache License 2.0`_ or `GPL 2`_.
+
+Installation
+============
+
+Check the current version of Adminer at `GitHub <https://github.com/vrana/adminer/releases>`_ and go the the folder where you want Adminer to be installed
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ VERSION=4.7.8
+  [isabell@stardust ~]$ cd ...
+  [isabell@stardust ~]$
+
+Option 1: Install Adminer without Plugin Support
+------------------------------------------------
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ wget -O index.php https://github.com/vrana/adminer/releases/download/v$VERSION/adminer-$VERSION.php
```

Comment:
This will install adminer into the home directory, instead of the [document root](https://manual.uberspace.de/web-documentroot/). Maybe `-O html/index.php`?

Reply:
I changed the code to cd into the html directory first